src/screens/discover/components/DiscoverHome.js

```diff
@@ -38,7 +39,7 @@ export default function DiscoverHome() {
 
   const hasHardwareWallets = Object.keys(wallets || {}).filter(key => wallets[key].type === walletTypes.bluetooth).length > 0;
 
-  const cards = useMemo(() => getCardsForPlacement(name), [name, getCardsForPlacement]);
+  const cardIds = useMemo(() => getCardIdsForScreen(name), [name, getCardIdsForScreen]);
```

Comment:
`name` here is always the name of the discover screen route, right?

Is the intention for this to react to changes in the `cardIds` state?

Reply:
If so, I think that state doesn't need to be accessed at the top level of discover — could be accessed in `RemoteCardCarousel` or in a wrapper component around it.

Looks like right now `getCardIdsForScreen` isn't invoked in the selector, so it won't be reactive beyond its initial value determined in the `useMemo`, but maybe that's the intention?

Reply:
Yeah I think I can clean this up a bit. I was getting some weird spacing issues which is why this is even here to begin with, but I can probably change `Stack` to a `Box` or something

Reply:
Yeah `<Box gap={x} />` shouldn't have that spacing problem when `null` is rendered.

Think when we first built `Stack`, the `gap` option didn't exist - been meaning to update that implementation.